.github/workflows/ci-pipeline.yml

```
@@ -31,33 +31,25 @@ jobs:
       - uses: actions/checkout@v2
         
       - name: Build Production Image
-        run: docker build --target production --tag markrobertdunn/todo-app:$GITHUB_SHA .
+        run: docker build --target production --tag markrobertdunn/todo-app:$GITHUB_SHA --tag markrobertdunn/todo-app:latest .
 
       - name: Login to Docker
         run: docker login --username markrobertdunn --password ${{ secrets.DOCKERHUBKEY }}
 
       - name: Publish Production Image to Docker
         run: docker push markrobertdunn/todo-app:$GITHUB_SHA
 
+      - name: Publish latest Image to Docker
+        run: docker push markrobertdunn/todo-app:latest
+
+
   Production_Release:
     name: Release
     runs-on: ubuntu-latest
     needs: Production_Build
-    env: 
-      HEROKU_API_KEY: ${{secrets.HEROKU_API_KEY}}
+
     steps:
       - uses: actions/checkout@v2
 
-      - name: Build Production Image
-        run: docker build --target production --tag markrobertdunn/todo-app:$GITHUB_SHA .
-
-      - name: login to heroku
-        run: echo $HEROKU_API_KEY | docker login --username mark.dunn@amt-sybex.com --password-stdin  registry.heroku.com
-
-      - run: heroku container:login
-
-      - run: docker tag markrobertdunn/todo-app:$GITHUB_SHA registry.heroku.com/devops-todoapp/web
-
-      - run: docker push registry.heroku.com/devops-todoapp/web
-
-      - run: heroku container:release web -a devops-todoapp
+      - name: Pull to azure from dockerhub
+        run: curl -dH -X POST ${{secrets.WEBHOOK_URL}}
```

One nice addition here can be adding the [--fail](https://curl.se/docs/manpage.html#-f) flag, as otherwise curl reports success as long as a successful connection was made, even if the response reports a failure (e.g. a status code > 400):
```suggestion
        run: curl -dH --fail -X POST ${{secrets.WEBHOOK_URL}}
```